test/apps/ping/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/flynn/flynn/discoverd/client"
+	"github.com/flynn/flynn/pkg/shutdown"
+)
+
+const service = "ping-service"
+
+func main() {
+	defer shutdown.Exit()
+
+	port := os.Getenv("PORT")
+	addr := ":" + port
+
+	hb, err := discoverd.AddServiceAndRegister(service, addr)
```

Comment:
Ouch, totally missed that, I think I copied the boilerplate code from the TCP app...